openfecli/commands/gather.py

```diff
@@ -230,11 +230,12 @@ def gather(rootdir, output, report):
     The results reported depend on ``--report`` flag:
 
     \b
-    * 'dg' (default) reports the ligand and the results are the maximum
-      likelihood estimate of its absolute free, and the uncertainty in
-      that.
+    * 'dg' (default) reports the ligand and the results as the maximum
```

Comment:
Sorry @mikemhenry I overwrote this a bit, totally agreed it needed a change, just thought it might need a bit of an expansion on what you wrote.

@hannahbaumann does this seem sensible to you?

Reply:
Yes, I think this generally sounds good, I was just a little bit confused about the "likelihood estimate of its absolute free" part and also the comma before ,and  makes it look a little bit like the "from DDG replica averages and standard deviations" only belongs to the "uncertainty" part.
Maybe something like: 
"* 'dg' (default) reports the ligand and its absolute free energy and the associated uncertainty as the maximum likelihood estimate obtained from DDG replica averages and standard deviations." Or maybe that's even more confusing, I'm not sure =)

Reply:
thanks @hannahbaumann ! I've updated it accordingly